bigquery_etl/backfill/utils.py

```diff
@@ -6,25 +6,31 @@
 from typing import List
 
 import click
+from google.cloud import bigquery
+from google.cloud.exceptions import NotFound
 
 from bigquery_etl.backfill.parse import BACKFILL_FILE, Backfill
 from bigquery_etl.cli.utils import (
     paths_matching_name_pattern,
     qualified_table_name_matching,
 )
+from bigquery_etl.util import extract_from_query_path
 
+BACKFILL_DESTINATION_PROJECT = "moz-fx-data-shared-prod"
```

Comment:
If this means that backfills from any project will be pre-processed in shared-prod, wondering if that has implications that we should consider e.g. costs.

Reply:
We will consider `shared-prod` in the first phase and expand to other projects at a later stage.